core/src/game_state.rs

```diff
@@ -1158,7 +1158,7 @@ impl PlayPhase {
         msgs
     }
 
-    pub fn finish_game(&self) -> Result<(InitializePhase, Vec<MessageVariant>), Error> {
+    pub fn finish_game(&mut self) -> Result<(InitializePhase, Vec<MessageVariant>), Error> {
```

Comment:
Hm. This shouldn’t mutate existing state? We are just sending a message, right?

Reply:
good catch, used for setting game state in the other branch